docs/proposals/template.md

```diff
@@ -0,0 +1,39 @@
+# Proposal Template
```

Comment:
Any particular reason for this template?

Asking as I had a [proposal](https://github.com/Kuadrant/multicluster-gateway-controller/blob/milestone-3/docs/milestone2/status_aggregation.md) on main that wasn't in the right directory and is now gone from main.
I'm going to PR it back and wondering what the intention behind these headings is.

Reply:
Copied from kcp-glbc https://github.com/kcp-dev/kcp-glbc/tree/main/docs/proposals

Reply:
I don't particularly care if we keep this template or not, but i do think it would be good to keep all all proposal docs in the same place.

Reply:
Agreed.